lib/jsdom/living/nodes/Element-impl.js

```diff
@@ -99,6 +99,7 @@ class ElementImpl extends NodeImpl {
     }
   }
 
+  /* eslint-disable-next-line jsdom-internal/hook-super-invocation */
```

Comment:
Disable eslint rule on top hooks that don't require super invocation.

Reply:
It seems like it'd be better if you stated the class names of the top-hierarchy classes for each hook in the config file; then these wouldn't be necessary.